platform/containers/kubernetes-distro/install.sh

```diff
@@ -16,25 +16,19 @@ apt-get update && apt-get install -y \
     ssh \
     git \
     vim \
-    python-pip \
-    && rm -rf /var/lib/apt/lists/* && rm -rf /var/cache/apt
+    python-pip
 
 pip install -r /tmp/requirements.txt
 pip install awscli
 
 curl -fSL "https://${DOCKER_BUCKET}/builds/Linux/x86_64/docker-$DOCKER_VERSION.tgz" -o docker.tgz \
     && echo "${DOCKER_SHA256} *docker.tgz" | sha256sum -c - \
     && tar -xzvf docker.tgz \
-    && mv docker/* /usr/local/bin/ \
-    && rmdir docker \
+    && mv docker/docker /usr/local/bin/ \
```

Comment:
we can remove entire docker binary as well as we are not calling docker to upgrade (another 20M reduce?)

Reply:
Docker client is such a handy tool that I'd prefer to keep. We are not worrying about extra 20MB now. There are other ways to get rid of 20MB.

Reply:
but we don't have docker sock inside the container so shy keeping it?